docs/textinput.md

```diff
@@ -437,7 +437,11 @@ Limits the maximum number of characters that can be entered. Use this instead of
 
 ### `multiline`
 
-If `true`, the text input can be multiple lines. The default value is `false`. It is important to note that this aligns the text to the top on iOS, and centers it on Android. Use with `textAlignVertical` set to `top` for the same behavior in both platforms.
+If `true`, the text input can be multiple lines. The default value is `false`. 
```

Comment:
```suggestion
If `true`, the text input can be multiple lines. The default value is `false`.
```

I think that this space annoy lint, however I was not able to directly commit to your branch to confirm that.